rq/job.py

```diff
@@ -393,6 +400,32 @@ def dependents_key(self):
         """The Redis key that is used to store job dependents hash under."""
         return self.dependents_key_for(self.id)
 
+    @property
+    def dependencies_key(self):
+        return '{0}:{1}:dependencies'.format(self.redis_job_namespace_prefix, self.id)
+
+    def fetch_dependencies(self, watch=False, pipeline=None):
+        """
+        Fetch all of a job's dependencies. If a pipeline is supplied, and
+        watch is true, then set WATCH on all the keys of all dependencies.
+
+        Returned jobs will use self's connection, not the pipeline supplied.
+        """
+        if pipeline is None:
+            pipeline = pipeline or self.connection
```

Comment:
Should this be `pipeline = self.connection`?

Reply:
:+1: 71a427279f90c5ab566ba741467ad2413994bc75